files/pull-sandbox-image.sh

```diff
@@ -2,6 +2,13 @@
 
 ### fetching sandbox image from /etc/containerd/config.toml
 sandbox_image=$(awk -F'[ ="]+' '$1 == "sandbox_image" { print $2 }' /etc/containerd/config.toml)
+
+### Short-circuit fetching sandbox image if its already present
+if [[ $(sudo ctr -n k8s.io images check name=="$sandbox_image") != "" ]]; then
```

Comment:
I think this check would always be `true`, because `ctr` always prints the headers. It also doesn't seem to indicate with the exit code whether any images matched the filters 👎 

So your `grep` solution is probably as good as anything

Reply:
Does it?

I tested this out. They don't print anything if there is no image

```
sh-4.2$ sudo ctr -n k8s.io images check name==602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pause:3.5
REF                                                        TYPE                                                      DIGEST                                                                  STATUS         SIZE                UNPACKED
602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pause:3.5 application/vnd.docker.distribution.manifest.list.v2+json sha256:529cf6b1b6e5b76e901abc43aee825badbd93f9c5ee5f1e316d46a83abbce5a2 complete (2/2) 290.5 KiB/290.5 KiB true
sh-4.2$ sudo ctr -n k8s.io images check name==602401143452.dkr.ecr.us-west-2.amazonaws.com/eks/pause:3.3
sh-4.2$
```

Reply:
With:
```
> ctr --version
ctr github.com/containerd/containerd 1.4.13
```
I see this on stdout:
```
> sudo ctr -n k8s.io images check name==foo 2> /dev/null
REF TYPE DIGEST STATUS SIZE UNPACKED 
```